### ISSUE TITLE
Allow users to see what starter packs they've been added to, and option to remove themselves

### ISSUE THREAD

**Issue by xenianth** [bluesky-social/social-app#5493]
**Is your feature request related to a problem? Please describe.**

the idea of starter packs is great, but there is no control given to users added to them. we aren't notified when we're added to one, we cannot see which packs we've been added to without third-party tools, and we cannot remove ourselves from packs. this keeps the door open to abuse.

i have personally been added to so many packs comprised of random accounts, as well as general art packs (that also include kidlit artists and the like), and as a body horror artist i really only want to be added to horror-specific ones, if any at all. i've been inundated with followers from these packs who have no interest in horror art, are minors, or are just bots. it's not only artificially inflating my follower count, it's causing huge fluctuations as so many of these people realize my content and leave. the leaving isn't bad, but the "50+ new followers... oh wait they're all gone" is highly annoying and extra frustrating as i have no control over what packs i'm added to.

**Describe the solution you'd like**

please allow us to consent to being added to packs or not. give us the ability to remove ourselves. notify us when we're added to a pack. allow us to actually see what packs we're in.

**Describe alternatives you've considered**

i have a line in my bio separated out clearly that states to not add me to any non-horror-art-related packs. this does nothing as no one reads or they don't care.

**Comment by MajorMcDoom** [comment]
Related: #5066

**Comment by nul800sebastiaan** [comment]
This is an important safety concern as noted in #5905 as well, please please allow us to at least consent to being on a starter pack and (now that it's already too late) remove ourselves from starter packs.

**Comment by cooljeanius** [comment]
Apparently ClearSky allows the viewing part at least, but I don't want to use ClearSky, I'd rather just do it from within BlueSky itself

**Comment by daffhunt** [comment]
Agreed, I came here to post this issue. I am getting flooded with unwanted follows all from accounts of a similar nature, and it's impossible to do anything about it. There should be options to deal with this. The most basic form would be a setting "Don't allow me to be added to lists or starter packs", a bit nuclear but good for people who want to grow their account organically. The second should be a way to review which lists you have been added to and remove yourself. Ideally also a notification that someone is trying to add you to a list with the option to decline. It would also be nice if follows arising from a list said "followed via list xxx"

**Comment by Leseratte10** [comment]
"Don't allow me to be added to starter packs" is a good idea. 

"Don't allow me to be added to lists" is a terrible idea. Lists are a core component for content moderation and for customizing the feed the way a user likes it. The very point of a moderation list, which is even described in the Bluesky app itself, is to be a public list of users for batch-muting or batch-blocking. If I don't want to hear about crypto, I can mute a mod list of people who commonly post crypto stuff. If I don't want to hear about politics, I can mute a mod list of people who commonly post about politics. And so on. 

The whole point of a moderation list is to mute or block people. Nobody likes being muted or blocked, but being able to mute or block chunks of people that are spamming your feed with posts you don't want to see drastically improves the usefulness of your feed. If you give people the option not to be added to a moderation list, you'll effectively make moderation lists completely useless. 

The only thing I'd say might be moderately useful might be "Don't show to the public which lists I'm a member of", but I'm not sure if that can even be implemented.

**Comment by daffhunt** [comment]
It is far too open to abuse, though. What is to stop someone starting a moderation list purporting to be one thing, building a large group of users who block via that list, and then one day adding their real victims who then get blocked by a great number of users simultaneously? The idea that owners of moderation lists can all be trusted to be acting in good faith is naïve and will surely bite badly at some point.

The fact that anyone can see which lists you have been added to is also open to abuse - someone could add you to a list with a title that suggests you have repugnant views (or worse) - causing reputational damage and possible pile-ons.

**Comment by Leseratte10** [comment]
Nothing is stopping anyone from starting such a moderation list. But I doubt a list created by a random dude is going to get massive traction. You'd probably need to spend months to curate such a list to get a great deal of people following it, just to get all that credibility thrown out the window when people figure out what you're up to and your account gets banned. The Bluesky team can still view lists and ban owners if someone decides to abuse their power. 

Sure, not all moderation list owners can be trusted. But it's still up to the users as to if they want to use moderation lists, and if so, which ones. And I assume that there will be a bunch of lists being shared (both on Bluesky and on other platforms like the subreddit) that are maintained by trusted people. Every list is public, so I'd assume that if a bunch of Democrats ended up on the MAGA list, for example, people would notice fairly quickly.

As for everyone seeing which lists you are on - correct, everyone can see that. Including you, so if you get added to a list that you think you don't belong on, you'd probably be able to report that list as well. Unless you have "Trump 2025 MAGA hooray!!" on your profile and don't think you belong on the MAGA list ...

I'd assume that the BlueSky developers have thought of that issue when they introduced public moderation lists. Yes, there will be some kind of abuse of these lists that can hopefully be worked around by user reports of such abuse happening. But giving users the ability to prevent their account from being added to lists completely negates the point of these lists. 

> someone could add you to a list with a title that suggests you have repugnant views (or worse) - causing reputational damage and possible pile-ons

Yes, just like someone can post on BlueSky or on Twitter "User X is an a-hole or has repugnant views", which would hopefully be dealt with when the post (or the list) gets reported.

To summarize - moderation lists will not work if you can remove yourself from such a list or prevent yourself from being added to one. It would make them entirely 100% useless - so the BlueSky team needs to decide, do we want mod lists (then there must be a way to report malicious lists or malicious entries but there shouldn't be a way for someone to remove themselves from one), or don't we want mod lists.

**Comment by xenianth** [comment]
this topic is about Starter Packs only. please keep this issue on topic so it can be properly addressed.
